syntax = "proto3";

import "google/protobuf/any.proto";

message ServerMessage {
	google.protobuf.Any message = 1;
}
